style.css

```diff
@@ -4821,6 +4821,20 @@ a.to-the-top > * {
 	.pagination-single .next-post .arrow {
 		margin: 0 0 0 2rem;
 	}
+	
+	.pagination-single-inner .next-post .arrow{
+		/*rtl:ignore*/
+		transform: rotate(-180deg);
+		-ms-transform: rotate(-180deg); /* IE 9 */
+  		-webkit-transform: rotate(-180deg); /* Safari */
+	}
+
+	.pagination-single-inner .previous-post .arrow{
+		/*rtl:ignore*/
+		transform: rotate(180deg);
+		-ms-transform: rotate(180deg); /* IE 9 */
+  		-webkit-transform: rotate(180deg); /* Safari */
+	}
```

Comment:
@Swapnildhanrale Looks almost perfect! I missed that the autoprefixer keeps adding `-ms-transform` and `-webkit-transform`. `/*rtl:ignore*/` only works for a single line, as you can tell above. In this case, we should use `/*rtl:begin:ignore*/` and `/*rtl:end:ignore*/` instead of `/*rtl:ignore*/`.

To ensure that the other statements don't end up in `style-rtl.css` can you apply the following changes.

Before:

```
	.pagination-single-inner .next-post .arrow{
		/*rtl:ignore*/
		transform: rotate(-180deg);
		-ms-transform: rotate(-180deg); /* IE 9 */
  		-webkit-transform: rotate(-180deg); /* Safari */
	}

	.pagination-single-inner .previous-post .arrow{
		/*rtl:ignore*/
		transform: rotate(180deg);
		-ms-transform: rotate(180deg); /* IE 9 */
  		-webkit-transform: rotate(180deg); /* Safari */
	}
```

After:

```
	.pagination-single-inner .next-post .arrow{
		/*rtl:begin:ignore*/
		transform: rotate(-180deg);
		-ms-transform: rotate(-180deg); /* IE 9 */
  		-webkit-transform: rotate(-180deg); /* Safari */
		/*rtl:end:ignore*/
	}

	.pagination-single-inner .previous-post .arrow{
		/*rtl:begin:ignore*/
		transform: rotate(180deg);
		-ms-transform: rotate(180deg); /* IE 9 */
  		-webkit-transform: rotate(180deg); /* Safari */
		/*rtl:end:ignore*/
	}
```

You can verify that your PR works if both `style.css` and `style-rtl.css` are showing the identical code. 😀

Reply:
Oh, and in https://travis-ci.com/WordPress/twentytwenty/jobs/244158564 Travis CI shows the following errors:

```
style.css
4824:1   ✖  Unexpected whitespace at end of no-eol-whitespace line                                                              
4825:43  ✖  Expected single space before "{"  block-opening-brace-space-before
4826:3   ✖  Expected empty line before comment-empty-line-before comment                                                           
4829:5   ✖  Expected indentation of 2 tabs indentation                     
4832:47  ✖  Expected single space before "{"  block-opening-brace-space-before
4833:3   ✖  Expected empty line before comment-empty-line-before comment                                                           
4836:5   ✖  Expected indentation of 2 tabs indentation
```

It's handy if you install [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) and [WordPress Coding Standards for PHP_CodeSniffer](https://github.com/WordPress/WordPress-Coding-Standards) on your computer to run these checks before uploading your changes. 😉